core/string/fuzzy_search.cpp

```diff
@@ -64,28 +64,28 @@ Ref<FuzzySearchResult> new_search_result() {
 	return result;
 }
 
-bool is_word_boundary(const String &str, const int index) {
+bool is_word_boundary(const String &str, int index) {
```

Comment:
There's nothing much wrong about passing  by const value but it's redundant and offers nothing so in C++ culture we just don't do it.